### DISCUSSION TITLE
进展标签的颜色有什么用呢？

### DISCUSSION THREAD

**Discussion by Yolkzz** [WatsonYao/nian_app#122] [Q&A] [answered]
不太明白怎么使用进展标签颜色这个功能，设置了进展标签的颜色后，在记本内选择标签时并没有显示相应的颜色。请问这个功能是怎么使用的呢？

**Comment by WatsonYao** [comment, marked as answer]
![image](https://github.com/user-attachments/assets/99a91dc5-9ac3-4f51-9717-f2d3c522fd80)
